### PR TITLE
[WFLY-7432] Elytron key-store requires set credential-reference which is wrongly marked as optional

### PR DESCRIPTION
**Pull request by pskopek**
https://issues.jboss.org/browse/WFLY-7432
https://issues.jboss.org/browse/JBEAP-6748

### CODE REVIEW

File: controller/src/main/java/org/jboss/as/controller/security/CredentialReference.java [hunk 1]
```diff
@@ -211,6 +207,22 @@ public static ObjectTypeAttributeDefinition getAttributeDefinition() {
     }
 
     /**
+     * Get the attribute builder for credential-reference attribute without default characteristics.
```
Comment (bstansberry) [inline controller/src/main/java/org/jboss/as/controller/security/CredentialReference.java]: s/without/with ?

**Comment by pskopek** [comment]
I have changed the description.

**Comment by bstansberry** [comment]
retest this please

Hard to imagine this PR broke 133 tests. ;)

**Comment by kabir** [comment]
Retest this please

**Pull request merged**